static/src/javascripts-legacy/projects/common/modules/tailor/tailor.js

```diff
@@ -9,8 +9,22 @@ define([
         });
     }
 
+    function getSuggestions(browserId) {
+        return fetchJson('https://tailor.guardianapis.com/suggestions?browserId=' + browserId, {
+            method: 'get'
+        });
+    }
+
+    function getRegularStatus(browserID) {
+        return getSuggestions(browserID).then(function(res) {
+            return res.userDataForClient.regular;
```

Comment:
just as defensive thing it could be worth using `try`/`catch` here and returning false if we don't have the right response from tailor